test/exprtest/blas1_axpy_copy_test.cpp

```diff
@@ -0,0 +1,133 @@
+/***************************************************************************
+ *
+ *  @license
+ *  Copyright (C) Codeplay Software Limited
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  For your convenience, a copy of the License has been included in this
+ *  repository.
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  SYCL-BLAS: BLAS implementation using SYCL
+ *
+ *  @filename blas1_axpy_copy_test.cpp
+ *
+ **************************************************************************/
+#include "blas_test.hpp"
+#include "sycl_blas.hpp"
+typedef ::testing::Types<blas_test_float<>, blas_test_double<> > BlasTypes;
+
+TYPED_TEST_CASE(BLAS_Test, BlasTypes);
```

Comment:
The test style you use is old. Please use the new interface test so that the code can be coherent. you can look at the blas1 test interface

Reply:
Updated